README.md

```diff
@@ -114,7 +112,7 @@ After you're finished please remove all the comments and instructions!
 
 > Add a link to your deployed project.
 
-- [Live Demo Link](<replace-with-your-deployment-URL>)
+- [Live Demo Link](replace-with-your-deployment-URL)
```

Comment:
- [ ] Please delete the template placeholder texts still present in some sections of your readme, this will make your readme more professional 
- [ ] Kindly remove the live demo link it is broken this will make your readme more professional